code-challenges.js

```diff
@@ -46,4 +101,15 @@ const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
 const cohortList = ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
 // Expected output: ["Delta", "Echo", "Foxtrot"]
 
-// Explain your code:
+// CREATE a function that takes in an array
+const indexFinder = (array) => {
+  // SEARCH for the values at the 3rd, 4th, and 5th, indexes
+  // RETURN the values at the 3rd, 4th, and 5th indexes
+  return array.slice(3, 6)
+}
+console.log(indexFinder(stockExchange));//outputs: [42, 5, -5]
+console.log(indexFinder(cohortList));//outputs: ['Delta', 'Echo', 'Foxtrot'] 
+
+
+
+// Explain your code: I created a function called indexFinder. Setting the input parameter to array. Then I returned the parameter and utilized the .slice method to extract the values at the third, fourth, and fifth index in the array. Setting the starting value to the value of the first index I want to extract. Then setting the second value to the index after the last index I'm looking to extract telling it where to stop. Calling the function and using the provided test variables to access the arrays stored within the output matches the expected output.
```

Comment:
great approach!  Push yourself to refactor using .filter - you can do it!  It will help you continue to learn. :)